docs/logging-stdlib-usage.rst

```diff
@@ -0,0 +1,71 @@
+Integration with Python logging module
+--------------------------------------
+
+
+It's possible to tie the Python :mod:`logging` module directly into Google Cloud Logging. To use it,
+create a :class:`CloudLoggingHandler <google.cloud.logging.CloudLoggingHandler>` instance from your
```

Comment:
@dhermes I didn't want to touch the text, just get it out of the way.  @waprin says he is busy modifying it anyway.